examples/mutators/set-label/v0_1_0/advanced/README.md

```diff
@@ -32,7 +32,7 @@ added to field `.spec.selector.labels` as well. We specify it in field
 Get the example config and try it out by running the following commands:
```

Comment:
is there a reason the directory uses `v0_1_0` instead of `v0.1.0`?

Reply:
Initially I thought it will cause problem on Windows. But it appears not to be true after googling it.
I can change it to use dot.